### PR TITLE
Make mannmelter less spamy

### PR DESCRIPTION
**Pull request by Dencube**
Higher damage cap+damage per hit but crits become minicrits and lower fire rate cap (DPS is around the same)

### CODE REVIEW

**Comment by Dencube** [comment]
Lowered damage on hit to 15%

**Comment by Dencube** [comment]
Changed my mind. Made damage on hit to 20%, lowered damage cap to 300%

File: addons/sourcemod/configs/vsh/vsh.cfg [hunk 1]
```diff
@@ -1131,8 +1131,8 @@
 		
 		"595"	//The Manmelter
 		{
-			"desp"			"Manmelter: {positive}+10% damage and +10% firing speed on hit (capped at +200% damage and +88% firing speed)"
-			"attrib"		"2 ; 1.0 ; 6 ; 1.0"	//This is so the stats can be modified
+			"desp"			"Manmelter: {positive}+20% damage and +10% firing speed on hit (capped at +300% damage and +80% firing speed), {negative}crits to minicrits"
```
Comment (FortyTwoFortyTwo) [inline addons/sourcemod/configs/vsh/vsh.cfg:1134]: `crit become minicrits` would be a better wording for it

Comment (Dencube) [reply addons/sourcemod/configs/vsh/vsh.cfg]: done

**Review by FortyTwoFortyTwo — APPROVED** [review]

**Pull request merged**